dockertest/subtest.py

```diff
@@ -19,16 +19,58 @@
 from ConfigParser import Error
 from autotest.client.shared.error import TestError, TestNAError
 from autotest.client.shared.version import get_version
-from autotest.client import test
+from autotest.client import test, utils
 import version
 import config
 import subtestbase
+from docker_daemon import which_docker
 from xceptions import DockerTestFail
 from xceptions import DockerTestNAError
 from xceptions import DockerTestError
 from xceptions import DockerSubSubtestNAError
 
 
+MEMO_CACHE = {}
```

Comment:
Constants are okay, but global variables in modules can lead to complicated synchronization and unittesting problems down the line.

This is fine for a POC.  But consider if there's a better way/place to store this temporary stuff.    If the data can't be encoded in an instance somewhere (singleton pattern comes to mind), the easiest future-proof solution is to make it private (single leading underscore) and write a function that returns it when called.